### PR TITLE
Added subtract function

### PR DESCRIPTION
**Pull request by edifofona**

### CODE REVIEW

File: pythoncalculator/subtract.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```
Comment (carolinedix) [inline pythoncalculator/subtract.py]: I think there might be a typo here...
```suggestion
    return x - y
```

**Review by carolinedix — CHANGES_REQUESTED** [review]
Thanks @EdifofonA This looks really useful, but I think there might be a typo in the subtract function.

File: pythoncalculator/subtract.py (hunk 1 above)
Comment (edifofona) [reply pythoncalculator/subtract.py]: Thanks @carolinedix, will fix

**Review by carolinedix — APPROVED** [review]
Great, thanks @EdifofonA Looks good to me!

**Pull request merged**